kani-compiler/src/args.rs

```diff
@@ -69,11 +82,12 @@ pub struct Arguments {
     /// Pass the kani version to the compiler to ensure cache coherence.
     check_version: Option<String>,
     #[clap(long)]
-    /// A legacy flag that is now ignored.
-    goto_c: bool,
-    /// Enable specific checks.
-    #[clap(long)]
     pub ub_check: Vec<ExtraChecks>,
+    #[clap(long = "backend")]
+    pub backend: Option<BackendOption>,
+    /// Print the final LLBC file to stdout.
+    #[clap(long)]
+    pub print_llbc: bool,
```

Comment:
Why would we print to the stdout?

Reply:
This is for testing purposes as part of the `expected` test suite. Since Aeneas is not yet called by the driver, the only option I could think of to test the code is to check that the LLBC is as expected.

Reply:
I think this has to at least be marked as a hidden option, since it is not a behavior we want to keep. This should also be documented as a temporary option, and that it requires `-Z aenas`.

For these things, we usually use the `script-based-pre` tests, but depending on the number of tests you want to add, it can become very painful to maintain. So I'll leave it up to you.

Reply:
> I think this has to at least be marked as a hidden option, since it is not a behavior we want to keep.

I recall trying to hide the option, but I couldn't find a way to do that. I'll check if that's possible.

> and that it requires -Z aenas.

I added that to the option documentation. The option validation would already error out if the option is specified without `-Zaeneas`.

> For these things, we usually use the script-based-pre tests, but depending on the number of tests you want to add, it can become very painful to maintain. So I'll leave it up to you.

Yeah, I considered this option, but it was indeed going to be painful.

Reply:
The option is now hidden.